DNN Platform/Library/Entities/Urls/RewriterUtils.cs

```diff
@@ -35,7 +35,7 @@ namespace DotNetNuke.Entities.Urls
 {
     public class RewriterUtils
     {
-        internal static readonly Regex ServicesFrameworkRegex = new Regex("DesktopModules/.+/API/", RegexOptions.Compiled);
+        internal static readonly Regex ServicesFrameworkRegex = new Regex("desktopModules/.+/api/|api/.+/action/", RegexOptions.Compiled);
```

Comment:
Hi @galatrash-at-dnn , this change is try to resolve the issue which introduced in DNN-8347, the service module won't match the regex in any case. if RegexOptions.IgnoreCase doesn't affect performance, i will change it to RegexOptions.IgnoreCase, can you help me confirm? thanks,